src/options.cpp

```diff
@@ -141,6 +141,13 @@ Options::Options()
       list_devices{false},
       use_device_with_uuid{}
 {
+    const char* var;
+    var = getenv("VKMARK_WINDOW_SYSTEM_DIR");
```

Comment:
Please add `#include <cstdlib>` for `getenv` (it works now due to stdlib being a transitive dependency of some other header, but that may change in the future)

Reply:
Done